docs/README.md

```diff
@@ -54,3 +54,34 @@ API docs unless you're making changes to the doc comments in the lit-element sou
 To regenerate the API docs and rebuild the  site, run the following command:
 
 `npm run build-all`
+
+### Deployment
+
+To build and deploy a new version of the site:
+
+```bash
+npm run deploy
+```
+
+(You must have permission to update the App Engine project to deploy the site.)
+
+The site is staged at a new URL (for example, `https://20181207t151022-dot-polymer-lit-element.appspot.com`). When ready, use the App Engine console to migrate traffic to the new version.
+
+To push to a named App Engine version, specify the version on the command line:
```

Comment:
Dumb question, but why would we ever want to push to a specific version? Wouldn't we either stage at the temporary URL shown above, or push to production?

Reply:
Good question. I do it, for example, when I'm working on a significant new chunk of doc that I expect will take several revisions, and I want people to be able to review it at a stable URL, like:

`https://new-tooling-section-dot-polymer-lit-element.appspot.com`

I considered mentioning that in the PR--I'll go ahead and add that explanation.